src/model.ts

```diff
@@ -157,6 +157,9 @@ abstract class $Node {
     return values(this).flatMap(extractChildren)
   }
 
+  @cached
+  siblings(): List<Node> { return this.parent().children() }
```

Comment:
Yo excluiría al nodo receptor, onda:

`return this.parent().children().filter(node => node !== this)`

Reply:
sí, tenés razón